mobile/native/src/orderbook.rs

```diff
@@ -32,9 +32,7 @@ use tokio::sync::watch;
 use tokio_tungstenite::tungstenite;
 use uuid::Uuid;
 
-/// The reconnect timeout should be high enough for the coordinator to get ready after a restart. If
-/// we reconnect too early we may not be ready process messages which require DLC actions.
-const WS_RECONNECT_TIMEOUT: Duration = Duration::from_secs(5);
+const WS_RECONNECT_TIMEOUT: u64 = 200;
```

Comment:
🔧 I think it's better to keep the constant as a `Duration`. Then it can be used without having to know whether we wanted it to be seconds or milliseconds.